src/main/java/hw10/FileLogger.java

```diff
@@ -0,0 +1,75 @@
+package hw10;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+public class FileLogger {
+    private final FileLoggerConfiguration configuration;
+    private final DateTimeFormatter fileDateFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public FileLogger(FileLoggerConfiguration configuration) {
+        this.configuration = configuration;
+    }
+
+    public void debug(String message) {
+        Objects.requireNonNull(message);
+        log(LoggingLevel.DEBUG, message);
+    }
+
+    public void info(String message) {
+        Objects.requireNonNull(message);
+        log(LoggingLevel.INFO, message);
+    }
+
+    private void log(LoggingLevel level, String message) {
+        if (level.compareTo(FileLoggerConfiguration.getLoggingLevel()) < 0) {
```

Comment:
This `if (level.compareTo(FileLoggerConfiguration.getLoggingLevel()) < 0) {` should be removed. This method should not apply any checks of a logging level. This is a job for the methods `info` and `debug`.